### PR TITLE
Reset app before each spec

### PR DESCRIPTION
**Pull request by ivodimitrov**

### CODE REVIEW

**Comment by ivodimitrov** [comment]
[Reset app before each spec](https://trello.com/c/CK76NyJ3/157-reset-app-before-each-spec)

File: spec/ios/costs/item_actions/item_actions_spec.rb [hunk 1]
```diff
@@ -1,19 +1,19 @@
 # frozen_string_literal: true
 
-require 'spec_helper'
-
+# Link to test cases at TestRail:
```
Comment (ivarbanovv) [inline spec/ios/costs/item_actions/item_actions_spec.rb:3]: why do we need a link in the spec?

**Review by ivarbanovv — APPROVED** [review]
Looking great . Merge it

File: spec/ios/costs/item_actions/item_actions_spec.rb (hunk 1 above)
Comment (ivodimitrov) [reply spec/ios/costs/item_actions/item_actions_spec.rb:3]: Test cases could be found easier. If you do not like the idea, I will remove the comment ;)

Comment (ivodimitrov) [reply spec/ios/costs/item_actions/item_actions_spec.rb]: Removed comments.

**Pull request merged**